src/azul/indexer/__init__.py

```diff
@@ -352,6 +352,39 @@ def __len__(self):
         """
         return len(self.common) + self.partition
 
+    def __contains__(self, partition_prefix: str) -> bool:
+        """
+        Same as `partition_prefix in prefix.partition_prefixes()` but more
+        efficient. See also :meth:`partition_prefixes`.
+
+        >>> p0, p1, p2 = Prefix.parse('/0'), Prefix.parse('/1'), Prefix.parse('/2')
+        >>> 'a' in p0
+        False
+        >>> 'a' in p1
+        True
+        >>> 'a' in p2
+        False
+
+        >>> p1, p2, p3 = Prefix.parse('a/0'), Prefix.parse('a/1'), Prefix.parse('a/2')
+        >>> 'ab' in p1
+        False
+        >>> 'ab' in p2
+        True
+        >>> 'ab' in p3
+        False
+
+        >>> 'ab' in Prefix.parse('b/1')
+        False
+
+        >>> 'aQ' in Prefix.parse('a/1')
```

Comment:
```suggestion
        >>> 'aA' in Prefix.parse('a/1')
```
`Q` seems an odd choice since it's outside the range of hex digits. I suggest `A` (showing that capital letters aren't included) or `g` (showing that only hex digits are included).